win32ss/gdi/dib/dib16bpp.c

```diff
@@ -164,21 +225,51 @@ DIB_16BPP_BitBltSrcCopy(PBLTINFO BltInfo)
           DIB_16BPP_PutPixel(BltInfo->DestSurface, i, j,
             XLATEOBJ_iXlate(BltInfo->XlateSourceToDest, 1));
         }
-        sx++;
+        if (bLeftToRight)
+        {
+          sx--;
+        }
+        else
+        {
+          sx++;
+        }
```

Comment:
In all these files there are such similar constructs. It would be much better for readability to define and use a macro like this:
```c
#define ITERATE(var, decrCond, increment) \
    ((var) = (decrCond) ? ((var) - (increment)) : ((var) + (increment)))
```
so that all of your blocks that look like the one above could now be written with a single readable line like this:
```c
    INCREMENT(sx, bLeftToRight, 1);
```

Reply:
Fixed by changing this code the new DEC_OR_INC macro.